testing/php7-libpuzzle/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Developer: Frank DENIS <j at pureftpd.org>
+# Contributor: Andreas Sundqvist <andreas at forme.se>
+# Maintainer: Timofey Bychkov <boobooin@gmail.com>
+pkgname=php7-libpuzzle
+pkgver=0.11
+pkgrel=0
+pkgdesc="PHP module of the Puzzle library. Library is designed to find visually similar images."
+url="http://libpuzzle.pureftpd.org/project/libpuzzle"
+arch="all"
+license="BSD-2-Clause"
+depends="libpuzzle"
+makedepends="libpuzzle php7-dev"
+source="https://download.pureftpd.org/pub/pure-ftpd/misc/libpuzzle/releases/libpuzzle-$pkgver.tar.gz
+		10_php7_support.patch
+		"
+builddir="$srcdir/libpuzzle-$pkgver"
+options="!check" # Libpuzzle was developed on OpenBSD, and one of the tests doesn't pass because floating point numbers are rounded differently on Linux. It has no implications on usage.
```

Comment:
the oneliner should work for most of php extensions `make NO_INTERACTION=1 REPORT_EXIT_STATUS=1 SKIP_ONLINE_TESTS=1 test`
if there's no tests it should be like `php7 -d extension="$builddir"/modules/libpuzzle.so --ri libpuzzle`
The edge case when extra dependencies needed see `community/php7-pecl-msgpack`